packages/contracts/src/dollar/mocks/MockGovernanceToken.sol

```diff
@@ -0,0 +1,19 @@
+// contracts/GLDToken.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockGovernanceToken is ERC20 {
```

Comment:
What is the purpose of a mock governance token in the same repo with the real governance token? Why not just use the real one?

Reply:
> What is the purpose of a mock governance token in the same repo with the real governance token? Why not just use the real one?

Originally, I thought I need ERC20 contract with transfer() function, but just found that it can be removed